db/periods.csv

```diff
@@ -76,7 +76,7 @@ mongol_empire,1206,1294,20.5, ,l,,1206,1304
 maya_classic,250,952,22.1,maya_classic_c,l,,230,970
 maya_postclassic,948,1539,23.1,maya_postclassic_c,l,,930,1539
 inca_empire,1438,1533,34.3,inca_empire_c,l,,1438,1572
-gupta_empire,319,550,20,gupta_empire_c,r,,270,550
+gupta_empire,319,550,20.7,gupta_empire_c,r,,270,550
```

Comment:
More space for "Rylands Library Papyrus P52" instead of "Papyrus P52", but maybe looks better anyway?